R/extract.R

```diff
@@ -72,6 +120,10 @@ st_extract.stars = function(x, at, ..., bilinear = FALSE, time_column =
 			try_result = try(x0 <- st_as_stars(x, downsample = dim(x)/2), silent = TRUE)
 			lapply(x, function(y) do.call(abind, lapply(get_names(y), 
 				gdal_extract, pts, resampling)))
+		} else if (exists("x_agg")) {
+			# Allow time matching for lines and polygons using aggregate
+			x = st_normalize(st_upfront(x_agg))
+			lapply(seq_along(x), function(i) x[[i]]) 
```

Comment:
line 126 does nothing, right?

Reply:
it does, it saves the array to `m` when `x_agg` was calculated

Reply:
I mean, it does nothing to x; you could have written `{ x = st_normalize(st_upfront(x_agg)); x }`

Reply:
Well not really, right? It is extracting the values from the stars object so that it can then pass to: https://github.com/r-spatial/stars/blob/e273215cd761f600184a0034d4230c6eefe9303f/R/extract.R#L173